### PR TITLE
V8: Make sure that the color picker labels are always shown

### PR DESCRIPTION
**Pull request by kjac**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes https://github.com/umbraco/Umbraco-CMS/issues/5393

### Description

See #5393 for a description on how to reproduce and test this PR 😄 

When this PR is applied, the color picker property configuration behaves like this:

color-picker-labels-enabled

### CODE REVIEW

**Comment by emmaburstow** [comment]
Hey @kjac 

Thanks for the PR. We'll let you know if we have any questions or if we need you to make any changes.

Em

**Comment by poornimanayar** [comment]
Hi @kjac ,

Could you test these scenarios for me? Go to an existing colour picker which has labels turned on already. Can you see the textbox, to begin with?

Another scenario is the same datatype as above

1. turn off the labels, save and reload, textbox disappears (all good there)
2. Now turn it back on, save and reload, I cannot see the textbox.

Can you verify these for me? I could replicate this even in incognito. 

Poornima

**Comment by kjac** [comment]
@poornimanayar that's weird! It worked when I did the PR, as you can tell from the gif. But I can reproduce the error in your scenarios... I'll get on it.

**Comment by kjac** [comment]
@poornimanayar so the whole problem is about the "enable labels" toggle and the color picker (with the text label) are two separate property editors. Which really doesn't make sense, but we can't change that now. 

Anyway, the two property editors initialize in their own scopes, which means they can't see each other's data, and thus the color picker has no idea if the "enable labels" toggle is on or off to begin with. To solve this, the toggle emits an event for the color picker on load, and apparently the color picker needs a bit more time to initialize before it's ready to catch the event.

Long story short. I added a bit of delay in the event emitting. It's by no means ideal, but short of a rewrite of color picker, this is what we're stuck with 😒

**Comment by nul800sebastiaan** [comment]
It's not ideal, but better than before! Let's go with this for now.
